code/about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Penguins</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel='icon' href='/favicon.ico' type='image/x-icon'>
+  </head>
+  <body>
+
+  <header>
+    <h1>PENGUINS</h1>
+  </header>
+
+  <div class="main">
+    <h2 class="font-header">Impressum</h2>
+    <p>Text Sources: <br>
+      <a href="https://www.penguins-world.com" target="_blank">Penguins-World</a><br>
+      <a href="https://www.neaq.org/wp-content/uploads/2016/06/LEARN_3-5-2_Penguin_TeacherGuide.pdf" target="_blank">New England Aquarium Penguins Teacher Guide</a><br>
+      <a href="https://www.insider.com/penguins-things-you-never-knew-facts-2018-4#there-are-same-sex-penguin-couples-8" target="_blank">Insider Magazine</a><br>
+      <a href="https://ocean.si.edu/ocean-life/seabirds/penguins" target="_blank">Smithsonian Ocean</a><br>
+      <a href="https://en.wikipedia.org/wiki/Penguin" target="_blank">Wikipedia</a><br>
+      <br>
+      Image Sources: <br>
+      <a href="https://www.google.com/search?q=penguins&tbm=isch&ved=2ahUKEwiMioylgZnsAhXaSEEAHas9AlMQ2-cCegQIABAA&oq=penguins&gs_lcp=CgNpbWcQAzICCAAyAggAMgIIADICCAAyAggAMgIIADICCAAyAggAMgIIADICCABQuiNYuiNg3SVoAHAAeACAAWGIAWGSAQExmAEAoAEBqgELZ3dzLXdpei1pbWfAAQE&sclient=img&ei=Lb14X8y_LNqRhbIPq_uImAU&bih=644&biw=1156&client=safari" target="_blank">Google Images</a><br>
+      <a href="https://www.neaq.org/wp-content/uploads/2016/06/LEARN_3-5-2_Penguin_TeacherGuide.pdf" target="_blank">New England Aquarium Penguins Teacher Guide</a><br>
+      <br>
+      <img src="https://pro2-bar-s3-cdn-cf.myportfolio.com/de6dcdbe9232ae9e5d743948d532292d/173c32c7ce905f4d746a76e2_rw_3840.jpg?h=025c931bddc87336432259f01574fb7a" alt="Your at-a-Glance Guide to Penguins" style="height:220px;"><br>
+      Penguin Page created 09/2020 by Mirjam B. for <a href="https://codeexcursion.ch" target="_blank">Code Excursion</a> &#128039; </p>
+    </div>
+     
+    <div class="sidenav">
+      <a href="./index.html">Home</a>
+      <a href="./introduction.html">Introduction</a>
+      <a href="./species.html">Penguin Species</a>
+      <a href="./map.html">World Map</a>
+      <a href="./funfacts.html">Fun Facts</a>
+      <a href="./video.html">Video</a>
+      <a class="active" href="./about.html">Impressum</a>
```

Comment:
Nice that you use `class="active"` to highlight the active link and I really like the sidenav! 🌟

A minor suggestion is to move up the sidenav a little bit (to move up the last box "Impressum" a little bit).